Three/include/CGAL/Three/Viewer_interface.h

```diff
@@ -99,8 +99,12 @@ class VIEWER_EXPORT Viewer_interface : public QGLViewer, public QOpenGLFunctions
   PFNGLDRAWARRAYSINSTANCEDARBPROC glDrawArraysInstanced;
   //!Allows OpenGL 2.1 context to get access to glVertexAttribDivisor.
   PFNGLVERTEXATTRIBDIVISORARBPROC glVertexAttribDivisor;
-  //!Allows OpenGL 2.1 context to get access to gkFrameBufferTexture2D.
+  //!Allows OpenGL 2.1 context to get access to glFrameBufferTexture2D.
   PFNGLFRAMEBUFFERTEXTURE2DEXTPROC glFramebufferTexture2D;
+  enum filter {
+   LINEAR=0,
+   NEAREST
+  };
   //!@returns true if glVertexAttribDivisor, and glDrawArraysInstanced are found.
```

Comment:
The fix doesn't work anyway.

2016-01-26 15:16 GMT+01:00 Laurent Rineau <notifications@github.com>:

> In Three/include/CGAL/Three/Viewer_interface.h
> <https://github.com/CGAL/cgal/pull/677#discussion_r50839129>:
>
> >    PFNGLFRAMEBUFFERTEXTURE2DEXTPROC glFramebufferTexture2D;
> > +  enum filter {
> > +   LINEAR=0,
> > +   NEAREST
> > +  };
> >    //!@returns true if glVertexAttribDivisor, and glDrawArraysInstanced are found.
>
> The enum filter seems unused!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CGAL/cgal/pull/677/files#r50839129>.
>